data/json/overmap/overmap_terrain/overmap_terrain.json

```diff
@@ -15,6 +15,14 @@
     "color": "dark_gray",
     "flags": [ "NO_ROTATE", "SHOULD_NOT_SPAWN" ]
   },
+  {
+    "type": "overmap_terrain",
+    "id": "explored",
```

Comment:
I think this should go in data/core, it's something the game always expects to exist.
We don't enforce anything going there, so it's not too important, though.

Reply:
Done in [816f430](https://github.com/CleverRaven/Cataclysm-DDA/pull/62760/commits/816f430435240c1545c8c98c2ac66db0c5c22ebb).